src/igvfd/schemas/changelogs/analysis_set.md

```diff
@@ -2,6 +2,8 @@
 
 ### Schema version 7
 
+* Restrict `publication_identifiers` to submission by admins only.
+* Add `publications`.
```

Comment:
These two entries should probably go under a new section "Minor changes since schema version 7"

Reply:
fixed.